### DISCUSSION TITLE
Surface pro 3 with Ubuntu 22.10

### DISCUSSION THREAD

**Discussion by ameer4** [linux-surface/linux-surface#1027] [Q&A] [answered]
I installed fresh Ubuntu 22.10 from usb, followed guide step by step.
I'm stuck at step :

`sudo apt install linux-image-surface linux-headers-surface iptsd libwacom-surface`

Terminal report:
`Some packages could not be installed. This may mean that you have
requested an impossible situation or if you are using the unstable
distribution that some required packages have not yet been created
or been moved out of Incoming.
The following information may help to resolve the situation:

The following packages have unmet dependencies:
 linux-headers-surface : Depends: linux-headers-6.1.2-surface (= 6.1.2-surface-1) but it is not going to be installed
 linux-image-surface : Depends: linux-image-6.1.2-surface (= 6.1.2-surface-1) but it is not going to be installed
E: Unable to correct problems, you have held broken packages.`

any idea?

**Comment by HarleySkin** [comment]
I installed fresh KDE Neo today and i have the same error:

The following packages have unmet dependencies:
linux-headers-surface : Depends: linux-headers-6.1.2-surface (= 6.1.2-surface-1) but it is not going to be installed
linux-image-surface : Depends: linux-image-6.1.2-surface (= 6.1.2-surface-1) but it is not going to be installed
E: Unable to correct problems, you have held broken packages.`

Funny is, on Sunday everything was going well with another fresh install.

**Comment by qzed** [comment, marked as answer]
Something went wrong with the latest build: Somehow the kernel/package revision got bumped at build time, which lead to the meta-package being built as version `6.1.2-surface-1` whereas the kernel package got built as version `6.1.2-surface-2`. The error you're seeing is the meta package looking for a kernel with version `6.1.2-surface-1`, but that doesn't exist.

I'm not sure why the version got bumped but I'm working on fixing that right now. Expect new packages in a couple of hours.

**Reply by ameer4** [reply]
Solved,
I have yet another issue: no sound from speakers!
I can see "built-in speakers" in sound settings.
But still no sound when trying to test it.

**Reply by NP-chaonay** [reply]
is there other speaker? and does non-builtin monitor or other speaker connected?

could you try install linux-firmware package, idk if that helps,

**Reply by ameer4** [reply]
No, no additional speakers except surface keyboard.
Keep in mind that fresh 22.10 still no sound.
I do feel Ubuntu wrongly selected other route for sound, like maybe activated external (display port?) as internal speaker. Need to test by connect surface to tv.

Side note: my surface has faulty wifi chip, running internet through external USB wifi.

**Reply by NP-chaonay** [reply]
@ameer4 

> maybe activated external (display port?) as internal speaker

if no connected to external monitor via display port or something like that, . Need to test by connect surface to tv.

then it cannot activate, because it is in disconnected state. So no need to test on that

> Keep in mind that fresh 22.10 still no sound. 

linux-firmware package is already installed?

> I can see "built-in speakers" in sound settings.

how about headphone, and does Youtube able to play without circle-spinning waiting indicator?

**Reply by ameer4** [reply]
I just booted Ubuntu to continue checking, to my surprise the sound works fine now, I really don't know how!
Weird..

Anyway, everything almost perfect for my little time with this machine, except camera are very slow & stuttering .
Is it normal?!
